README.md

```diff
@@ -143,18 +143,22 @@ You can find sample code for v2 in the following places:
 * Integration tests in this repo. They are located in the `it` directory under each service module, eg: [s3-integration-tests]
 
 ## Maintenance and Support for SDK Major Versions
-For information about maintenance and support for SDK major versions and their underlying dependencies, see the following in the AWS SDKs and Tools Shared Configuration and Credentials Reference Guide:
+For information about maintenance and support for SDK major versions and their underlying dependencies, see the following in the AWS SDKs and Tools Reference Guide:
 
 * [AWS SDKs and Tools Maintenance Policy][maintenance-policy]
 * [AWS SDKs and Tools Version Support Matrix][version-matrix]
 
+## Maintenance and Support for Java Versions
+
+We maintain full support on Long-Term Support(LTS) releases: Java 8, Java 11, and Java 17.
```

Comment:
Should we clarify what do we do when it's a non-LTS version? 
Something like "If it's a non-LTS Java version, cases will be analyzed but supported on a best-effort basis".

Reply:
Discussed offline and decided to keep it as-is